lib/galaxy/tool_shed/util/container_util.py

```diff
@@ -60,9 +60,9 @@ def get_components_from_key(key):
 def print_folders(pad, folder):
```

Comment:
This method seems to be unused, so it could actually make sense to remove it from here and lib/tool_shed/util/container_util.py

Reply:
Done.